cli/azd/pkg/infra/provisioning/bicep/prompt.go

```diff
@@ -81,6 +81,37 @@ func autoGenerate(parameter string, azdMetadata azure.AzdMetadata) (string, erro
 	return genValue, nil
 }
 
+// azureAiServicesLocations is the locations where Azure AI Services are available.
+// Acquired from Azure Portal on 3/31/2025
+var azureAiServicesLocations = []string{
+	"australiaeast",
+	"brazilsouth",
+	"canadacentral",
+	"canadaeast",
+	"centraluseuap",
+	"eastus",
+	"eastus2",
+	"eastus2euap",
+	"francecentral",
+	"germanywestcentral",
+	"japaneast",
+	"koreacentral",
+	"northcentralus",
+	"norwayeast",
+	"polandcentral",
+	"southafricanorth",
+	"southcentralus",
+	"southindia",
+	"southeastasia",
+	"swedencentral",
+	"switzerlandnorth",
+	"uaenorth",
+	"uksouth",
+	"westeurope",
+	"westus",
+	"westus3",
+}
```

Comment:
From the network logs, the AI Services Portal blade seems to be getting these locations from the [Resource Skus - List endpoint](https://learn.microsoft.com/en-us/rest/api/aiservices/accountmanagement/resource-skus/list?view=rest-aiservices-accountmanagement-2024-10-01&tabs=HTTP). Could we use that instead of hardcoding locations here? When testing on my sub, I also get 26 entries/locations where `kind=AIServices` that match the locations here.

I'm not sure if this was what you were referring to in the description - I didn't quite understand this part:

> While we found a new Endpoint or way to get the list of locations which supports Azure AI Services, we are adding the locations that the Azure Portal is providing currently for Azure AI Services creation.

Reply:
updated